packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -504,79 +502,9 @@ export default {
           },
         },
       });
-      this.insertContent();
       this.addEditorListeners();
     },
 
-    /**
-     * This function is necessary as tiptap doesn't render the content passed
-     * directly through `editor.commands.setContent` the content passed down to it
-     * should be already parsed. So We're parsing the elements into it's corresponding
-     * HTML version before setting it.
-     */
-    insertContent () {
-      this.parseMentions();
-      this.parseChannels();
-      this.parseEmojis();
-      this.editor.commands.setContent(this.internalValue, true);
-    },
-
-    parseEmojis () {
-      const matches = new Set(
-        [...this.value.matchAll(emojiRegex()), ...this.value.matchAll(emojiShortCodeRegex)]
-          .map(match => match[0].trim()),
-      );
-
-      if (!matches) return;
-
-      matches
-        .forEach(match => {
-          const emoji = codeToEmojiData(match);
-          if (!emoji) return;
-          this.internalValue = this.internalValue.replace(new RegExp(`${match}`, 'g'), `<emoji-component code="${emoji.shortname}"></emoji-component>`);
-        });
-    },
-
-    parseChannels () {
-      if (!this.channelSuggestion) return;
-
-      const suggestions = this.channelSuggestion.items({ query: '' });
-      const matches = [...this.value.matchAll(channelRegex)]
-        .filter(match => suggestions.some(({ id }) => id === match[1]));
-
-      if (!matches) return;
-
-      matches.forEach(match => {
-        const channel = suggestions.find(({ id }) => id === match[1]);
-        this.internalValue = this.internalValue.replace(
-          `#${match[1]}`,
-          /** The space at the beginning is important as tiptap removes that while rendering.
-           *  So if multiple mentions, channels or emojis are next to each other it will fail
-           */
-          ` <channel-component name="${channel.name}" id="${channel.id}"></channel-component>`,
-        );
-      });
-    },
-
-    parseMentions () {
-      if (!this.mentionSuggestion) return;
-
-      const suggestions = this.mentionSuggestion.items({ query: '' });
-      const matches = [...this.value.matchAll(mentionRegex)]
-        .filter(match => suggestions.some(({ id }) => id === match[1]));
-
-      if (!matches) return;
-
-      matches.forEach(match => {
-        const mention = suggestions.find(({ id }) => id === match[1]);
-        this.internalValue = this.internalValue.replace(`@${match[1]}`, ` <mention-component name="${mention.name}" id="${mention.id}"></mention-component>`);
-      });
-    },
-
-    destroyEditor () {
-      this.editor.destroy();
-    },
```

Comment:
We don't need this function anymore? or we weren't using it anywere?

Reply:
Hmm.. think I didn't mean to delete this.